plugins/sentinelone/help.md

```diff
@@ -1085,7 +1140,6 @@ This action is used to create a threat from an IOC event.
 |path|string|None|False|Path|None|path|
 
 Example input:
-
```

Comment:
Did you mean to remove all these new lines in the help.md?

Reply:
@cmcnally-r7 , yes it was intentional as it seemed to be throwing an issue during validation when I removed them.